src/open_inwoner/scss/components/UserFeed/UserFeed.scss

```diff
@@ -84,17 +84,12 @@
 
```

Comment:
I assume the empty status indicator is supposed to scale with the height of the indicator with text on the same row, to keep the cards aligned. So this is probably intentional, right? Just wanted to double check
![image](https://github.com/maykinmedia/open-inwoner/assets/29249171/9c4a0640-0579-4fa3-ba0c-0bda03100011)

Reply:
Oh dear, the Warning one on the right does not look good... Let's see if I can fix that for empty statuses.

Reply:
I think these below are the most common edge/weird cases for no-content/too-long content, but in reality there will likely always be a one-line status_text - if this 'marker' is not what we want for empty statuses, I will have to ask the designer next week:

<img width="1261" alt="Screenshot 2024-03-07 at 15 52 20" src="https://github.com/maykinmedia/open-inwoner/assets/118177951/18b36d84-1e74-4d6a-ad31-ea8c5b45cfef">